src/irmin/import.ml

```diff
@@ -22,10 +22,8 @@ type read_write = Perms.read_write
 
 (** {2 Lwt syntax} *)
 
-let ( >>= ) = Lwt.Infix.( >>= )
-let ( >|= ) = Lwt.Infix.( >|= )
```

Comment:
Let's stick with explicit imports from `Lwt.Infix`; it has a few extra operators we never use.

Reply:
fixed